Projects/Feature/Chat/Sources/Chat/ChatView.swift

```diff
@@ -34,8 +34,7 @@ public struct ChatView: View {
           }),
           centerTitle: "새 업무 체크리스트"
```

Comment:
```suggestion
          centerTitle: "프로젝트 추가"
```
엇 피그마 디자인이랑 다른 부분이 있네요!
수정해주시면 감사하겠씁니다!!

혹쉬 수정하실 때 `EnterKeywordView`에서 **업무 체크리스트 저장** -> **프로젝트 저장**으로 같이 수정 부탁드려도 될까요? 🙇‍♂️